### ISSUE TITLE
no autocomple

### ISSUE THREAD

**Issue by ccqgithub** [atom/autocomplete-emojis#14]
my mac osx, atom version 1.11.2,  autocomplete-emojis version 2.5.0,  no autocomple ……

and no error tips

**Comment by eqot** [comment]
This autocomplete does not always work.
This works in *.md, *.html or etc which is described at "Scopes" in README.md.

What kind of file did you edit?

**Comment by tribut** [comment]
Is there a way to add additional scopes locally somewhere?

**Comment by phanirithvij** [comment]
@eqot Yeah, why do normal emojis (not markdown) not work everywhere? I want to use them in `*.py, *.go, *.sh` files too, not just html.
